util/options.py

```diff
@@ -334,6 +334,20 @@ def rom_address_padding() -> bool:
     return opts.get("rom_address_padding", False)
 
 
+# Determines which ABI names to use for general purpose registers
+# Valid values: 'numeric', 'o32', 'n32', 'n64'
+def get_mips_abi_gpr() -> str:
+    return opts.get("mips_abi_gpr", "o32")
+
+
+# Determines which ABI names to use for floating point registers
+# Valid values: 'numeric', 'o32', 'n32', 'n64'
+# o32 is highly recommended, as it provides logically named registers for floating point instructions
+# For more info, see https://gist.github.com/EllipticEllipsis/27eef11205c7a59d8ea85632bc49224d
+def get_mips_abi_float_regs() -> str:
+    return opts.get("mips_abi_float_regs", "numeric")
```

Comment:
Why not o32 by default?

@EllipticEllipsis :smirk:

Reply:
lol

I would have considered changing the default if assemblers supported it by default, but you need to add those register names as macros manually, like this:
https://github.com/zeldaret/mm/blob/8049dec3393c4ade687dbdfd84d3ccde73168ab0/include/macro.inc#L42-L75

So changing this default would be a breaking change for people who just wants to update their splat. @ethteck what do you think about it?

Reply:
Yeah I'd prefer not to switch this out on everyone. Sorry... maybe for a later version once we evangelize the community enough :P

Reply:
If GAS supported them, by all means. But these are the current defaults for objdump anyway, which seems reason enough.